interview-questions.md

```diff
@@ -7,41 +7,41 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Rails follows an MVC pattern. What does that mean?
 
-  Your answer:
+  Your answer:MVC stans for, models, controllers, and views. its a pattern of designing used in rails.
 
-  Researched answer:
+  Researched answer: MVC, or model,controller,view pattern is a pattern that contains 3 core architecture. models is used for handling logi, and data. controllers are for handling user interface, and views is for handling the graphical user interface objects. 
 
 
 
 2. What is a relational database? Are there other kinds of databases?
 
-  Your answer:
+  Your answer: a relational database relates different models with each other user specific commands such as belongs to and has many.
 
-  Researched answer:
+  Researched answer:relational databases, also refered to as an SQL database as above they can use specific commands to relate information. Also there are non relational databases specifically not using tabular schema.
 
 
 
 3. What is object-relational mapping?
 
-  Your answer:
+  Your answer: ORM is used to relate incompatible systems using object oriented programming. 
 
-  Researched answer:
+  Researched answer:ORM in CS is a programming technique that converts data from a non compatible type and relates them using OOP this createes a virtual database for a system to work from. 
```

Comment:
Hey I have this Wikipedia page pulled up too! 

Remember answers have to be in your own words and cannot be copied from the internet!  

SENDING IT BACK!